search_log.go

```diff
@@ -136,20 +175,22 @@ func readLine(reader *bufio.Reader) (string, error) {
 	return string(line), nil
 }
 
-func readLastLine(file *os.File) string {
+// Read a line from the end of a file.
+// TODO: read byte by byte is low efficiency, we can improve it, for example, read 1024 bytes one time
```

Comment:
Can we refine by this way? (not sure whether it has some pitfalls.)

Reply:
Sure, we can add a todo here.